editor/import_dock.cpp

```diff
@@ -153,7 +158,11 @@ void ImportDock::_add_keep_import_option(const String &p_importer_name) {
 	import_as->add_separator();
 	import_as->add_item(TTR("Keep File (No Import)"));
 	import_as->set_item_metadata(-1, "keep");
+	import_as->add_item(TTR("Skip File (No Import)"));
```

Comment:
Should this be:
```suggestion
	import_as->add_item(TTR("Skip File (No Export)"));
```
?
Or "No Import nor Export" but it starts being quite lengthy.

Reply:
Not sure if `No Import` part is necessary, it's not adding any information. Maybe `Keep File (exported as is)` and `Skip File (not exported)`?

Reply:
Renamed options to: `Keep File (exported as is)` and `Skip File (not exported)`.

Also added some doc notes: https://github.com/godotengine/godot-docs/pull/9089